pkg/roll/run_update.go

```diff
@@ -182,12 +182,10 @@ ROLL_LOOP:
 					"new":  newNodes,
 					"next": next,
 				}).Infoln("Undergoing next update")
-				if u.DeletePods {
-					err = u.rcs.AddDesiredReplicas(u.OldRC, -next)
-					if err != nil {
-						u.logger.WithError(err).Errorln("Could not decrement old replica count")
-						break
-					}
+				err = u.rcs.AddDesiredReplicas(u.OldRC, -next)
```

Comment:
So this works because when the old RC is having its count reduced, it is also disabled, correct? Otherwise there's a risk that it could unschedule one of its active, not yet deployed-over instances.